dlclive/dlclive.py

```diff
@@ -478,3 +486,154 @@ def close(self):
         self.is_initialized = False
         if self.display is not None:
             self.display.destroy()
+
+
+class MultiAnimalDLCLive(DLCLive):
+    def __init__(
+        self,
+        model_path,
+        n_animals,
+        n_multibodyparts,
+        track_method: str = "box",
+        min_hits: int = 1,
+        max_age: int = 1,
+        sim_threshold: float = 0.6,
+        resize: Optional[float] = None,
+        convert2rgb: bool = True,
+        processor: Optional['Processor'] = None,
+        display: typing.Union[bool, Display] = False,
+        pcutoff: float = 0.5,
+        display_radius: int = 3,
+        display_cmap: str = "bmy",
```

Comment:
missing arguments from parent signature:
* `model_type`
* `precision`
* `tf_config`
* `cropping`
* `dynamic`

Any reason? We should try and make uniform API if possible.

Since these are all in the `DLCLive.PARAMETERS` tuple, this will cause `.parameterization` to error. This also removes control over the `process_frame` method.

Two approaches: add the missing arguments, or else remove all of them and accept a `**kwargs` that gets passed to the parent class.